test/Templates.Test/ByteOrderMarkTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Templates.Test
+{
+    public class ByteOrderMarkTest
+    {
+        private readonly ITestOutputHelper _output;
+
+        public ByteOrderMarkTest(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [Theory]
+        [InlineData("\\Microsoft.AspNetCore.SpaTemplates\\content")]
+        [InlineData("\\Microsoft.DotNet.Web.ProjectTemplates\\content")]
+        [InlineData("\\Microsoft.DotNet.Web.Spa.ProjectTemplates\\content")]
+        public void CheckForByteOrderMarkSpaTemplates(string path)
+        {
+            var currentDirectory = Directory.GetCurrentDirectory();
+            var srcDirectory = Path.GetFullPath(Path.Combine(currentDirectory, @"..\..\..\..\..\src"));
+            var directories = Directory.GetDirectories(srcDirectory + path, "*Sharp");
+            var checkFiles = false;
```

Comment:
Not sure I understand the semantics of this variable. Can we find a better name for it?

Reply:
Basically a flag that says the test output should be checked for files that have BOM characters.

Reply:
Isn't it saying the *result* of the test (success/failure), as opposed to whether something should be checked?